tidb-lightning/tidb-lightning-overview.md

```diff
@@ -44,3 +44,7 @@ TiDB Lightning 整体工作原理如下：
 如果需要导入的目标集群是 v3.x 或以下的版本，需要使用 Importer-backend 来完成数据的导入。在这个模式下，`tidb-lightning` 需要将解析的键值对通过 gRPC 发送给 `tikv-importer` 并由 `tikv-importer` 完成数据的导入。
 
 TiDB Lightning 还支持使用 TiDB-backend 作为后端导入数据：`tidb-lightning` 将数据转换为 `INSERT` 语句，然后直接在目标集群上执行这些语句。详见 [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md)。
+
+> **注意：**
+>
+> TiDB Lightning 不支持对有 TiFlash 副本的表导入数据。在导入数据之前请先清除 TiFlash 副本。你可以在导入完成之后再添加 TiFlash 副本。
```

Comment:
🤔 if so this is likely a regression. we've got a [test case](https://github.com/pingcap/br/blob/master/tests/lightning_tiflash/run.sh) to ensure Lightning can work when TiFlash is present, but it seems it is skipped on the CI.

Anyway it should be clarified that this only affects the Local Backend.

Reply:
My mistake. We can use Lightning with TiFlash after v4.0.6.